### ISSUE TITLE
Prepare scripts for training a classification model

### ISSUE THREAD

**Issue by mozharovsky** [formermagic/formerbox#4]
## Summary

Our current scripts only support language models pre-training. Which is a first step towards classification models built on top of a pre-trained language model (namely, RoBERTa LM). The next step is fine-tuning a language model on a downstream task (aka commit type classification).

This requires a new Pytorch-lightning module for classification pipeline along with specific data readers that support classification labels for each instance.

## Steps

- [ ] Pytorch-lightning Classification Module
- [ ] Classification Dataset (based on IndexedDataset)
- [ ] Classification Dataset Preprocessing
- [ ] Train and evaluate a sample model to validate correctness
